src/main/java/com/iota/iri/service/validation/TransactionValidator.java

```diff
@@ -210,148 +160,6 @@ public TransactionViewModel validateBytes(final byte[] bytes, int minWeightMagni
-        if(maxProcessedTransactions != Integer.MAX_VALUE) {
-            maxProcessedTransactions += analyzedHashes.size();
-        }
-        boolean solid = true;
-        final Queue<Hash> nonAnalyzedTransactions = new LinkedList<>(Collections.singleton(hash));
-        Hash hashPointer;
-        while ((hashPointer = nonAnalyzedTransactions.poll()) != null) {
-            if (!analyzedHashes.add(hashPointer)) {
-                continue;
-            }
-
-            if (analyzedHashes.size() >= maxProcessedTransactions) {
-                return false;
-            }
-
-            TransactionViewModel transaction = fromHash(tangle, hashPointer);
-            if (!transaction.isSolid() && !snapshotProvider.getInitialSnapshot().hasSolidEntryPoint(hashPointer)) {
-                if (transaction.getType() == PREFILLED_SLOT) {
-                    solid = false;
-
-                    if (!transactionRequester.isTransactionRequested(hashPointer)) {
-                        transactionRequester.requestTransaction(hashPointer);
-                        continue;
-                    }
-                } else {
-                    nonAnalyzedTransactions.offer(transaction.getTrunkTransactionHash());
-                    nonAnalyzedTransactions.offer(transaction.getBranchTransactionHash());
-                }
-            }
-        }
-        if (solid) {
-            updateSolidTransactions(tangle, snapshotProvider.getInitialSnapshot(), analyzedHashes);
-        }
-        analyzedHashes.clear();
-        return solid;
-    }
-
-    public void addSolidTransaction(Hash hash) {
-        synchronized (cascadeSync) {
-            if (useFirst.get()) {
-                newSolidTransactionsOne.add(hash);
-            } else {
-                newSolidTransactionsTwo.add(hash);
-            }
-        }
-    }
-
-    /**
-     * Creates a runnable that runs {@link #propagateSolidTransactions()} in a loop every {@value #SOLID_SLEEP_TIME} ms
-     * @return runnable that is not started
-     */
-    private Runnable spawnSolidTransactionsPropagation() {
-        return () -> {
-            while(!shuttingDown.get()) {
-                propagateSolidTransactions();
-                try {
-                    Thread.sleep(SOLID_SLEEP_TIME);
-                } catch (InterruptedException e) {
-                    // Ignoring InterruptedException. Do not use Thread.currentThread().interrupt() here.
-                    log.error("Thread was interrupted: ", e);
-                }
-            }
-        };
-    }
-
-    /**
-     * Iterates over all currently known solid transactions. For each solid transaction, we find
-     * its children (approvers) and try to quickly solidify them with {@link #quietQuickSetSolid}.
-     * If we manage to solidify the transactions, we add them to the solidification queue for a traversal by a later run.
-     */
-    @VisibleForTesting
-    void propagateSolidTransactions() {
```

Comment:
I thought you moved this to the new Solidifier...
Now I am noticing that it is not there..

I agree that this code should be changed/removed but why in this PR
See #1009. There was already a PR for it btw